datapackage.json

```diff
@@ -244,92 +286,110 @@
                         "format": "default"
                     },
                     {
-                        "name": "Provenance",
-                        "type": "string",
+                        "name": "Century Numeric",
+                        "type": "integer",
                         "format": "default"
                     },
                     {
-                        "name": "Link",
+                        "name": "Link to Digital Copy",
                         "type": "string",
                         "format": "default"
                     },
                     {
-                        "name": "Notes",
-                        "type": "string",
+                        "name": "Macomber Manuscript",
+                        "type": "boolean",
                         "format": "default"
                     },
                     {
-                        "name": "vHMML permalink (pending on 04/25/2020)",
-                        "type": "string",
+                        "name": "Delamarter Manuscript",
+                        "type": "boolean",
                         "format": "default"
                     },
                     {
-                        "name": "Columns per page",
+                        "name": "Total Pages",
                         "type": "integer",
                         "format": "default"
                     },
                     {
-                        "name": "Lines per column",
-                        "type": "string",
+                        "name": "Total Scans",
+                        "type": "integer",
                         "format": "default"
                     },
                     {
-                        "name": "Characters per line",
+                        "name": "Columns per page",
                         "type": "integer",
                         "format": "default"
                     },
                     {
-                        "name": "Hamburg MS ID",
+                        "name": "Line range per column",
                         "type": "string",
                         "format": "default"
                     },
                     {
-                        "name": "Latitude",
-                        "type": "number",
+                        "name": "Lines per column",
+                        "type": "string",
                         "format": "default"
                     },
                     {
-                        "name": "Longitude",
-                        "type": "number",
+                        "name": "Characters per line",
```

Comment:
While we're changing this, could we use consistent capitalization? (I think this one and the three above are the only ones that are different)